pages/reference-smart-account/owners/getThreshold.mdx

```diff
@@ -0,0 +1,36 @@
+import { Tabs } from 'nextra/components'
+
+# `getThreshold`
+
+Returns the number of required confirmations for a Safe transaction aka the threshold.
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
+        function getThreshold() external view returns (uint256)
+    }
+
+    contract Example {
+        function example() ... {
+            (uint256 threshold) = (ISafe safe).getThreshold(); // 1
```

Comment:
```suggestion
            (uint256 threshold) = (ISafe safe).getThreshold();
```
Could you remove the return values commented. The Returns section with its type should be enough. Here I don't find it very clear. It's also not used anywhere else.